### DISCUSSION TITLE
Welcome to esp32-smartdisplay Discussions!

### DISCUSSION THREAD

**Discussion by rzeldent** [rzeldent/esp32-smartdisplay#43] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by AxelHamburch** [comment]
Thanks for esp32-smartdisplay. 🙏 I use it for a Sunton 3.5" 320*480 Smart Display ESP32-3248S035 to realize an offline Lightning ⚡ Switch on Bitcoin and it works wonderfully. 🧡

https://github.com/AxelHamburch/offlineLNSwitch

**Reply by rzeldent** [reply]
Looks really nice!

**Reply by AxelHamburch** [reply]
Your seeed_xiao_esp32s3 works too. I couldn't manage it with the AI-Thinker ESP32-CAM. Always the problem with the bad RAM. I tried four pieces, but the same problem with all of them. The Xiao finally worked and I'm now building something nice with it. 😄
